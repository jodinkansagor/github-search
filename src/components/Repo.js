import React from 'react';
import PropTypes from 'prop-types';

const Repo = (repo) => (
  <section>
    <h1>{repo.name}</h1>
    <a href={repo.html_url}>Go to the Repo!</a>
  </section>
);

Repo.propTypes = {
  repo: PropTypes.any
};

export default Repo;
