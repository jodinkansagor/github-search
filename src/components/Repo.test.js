import React from 'react';
import { shallow } from 'enzyme';
import Repo from './Repo';

describe('App component', () => {
  it('renders App', () => {
    const wrapper = shallow(<Repo />);
    expect(wrapper).toMatchSnapshot();
  });
});
