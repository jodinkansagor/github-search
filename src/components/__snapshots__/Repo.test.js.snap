// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<section
  className="repo"
>
  <a>
    <h1 />
  </a>
</section>
`;
